### DISCUSSION TITLE
Задротов CSS не победить / You can't beat CSS nerds

### DISCUSSION THREAD

**Discussion by PavButsa** [cssbattle/forum#353] [General]
Ну и чё нам дали эти ежедневные цели с открытыми решениями? Все те же задроты, которые висели в топе, так там и висят. Из ежедневок ничему нельзя научиться, этих CSS-задротов невозможно разъебать. В топе новой битвы всё те же рожи.
А у меня бомбит, что я нихера не понимаю в их решениях!
Щас Илья Стрельцын сидит и ржёт, привет, Илюха!
/
So what's the use of those daily targets with open solutions? All nerds from leaderbords' tops remains there. You can't learn a thing from daily targets, you can't beat CSS neards. There are the same persons in the new battle's learderboard.
It pisses me off that I don't understand their solutions.
Ilya Streltsyn laughs right now, hi Ilya!

**Comment by chinchang** [comment]
I'll recommend joining our [Discord server](https://discord.gg/rvqvbcFM86) where lot of people discuss past day solutions and also put beautiful detailed explanations of how they did what they did! I have personally learnt a lot from them!
I am sure if you ask once in our Discord if anyone has learnt anything from Daily Targets, there will be something by everyone. :)

**Comment by SelenIT** [comment]
Спокуха, без паники / No panic! 😄

I learned many things myself from these targets, e.g. practical uses of CSS nesting or the fact that the background of the root element now ignores transformations. The latter enabled this solution of the 24/08/2023 target by @iamvdo which I'd never find myself:
```css
<style>*{ /* applies to both html and body */
   /* ensures that body is taller than 400px,
      so fills the viewport width when rotated */
  margin:-7%0;
  /* html is rotated by 45deg, but its background doesn't;
     body is rotated by 45+45=90deg along with its background */
  rotate:45deg;
  background:
      /* transparent circle with 120px radius in the blue area */
      radial-gradient(127q,#fff0 99.8%,#4F77FF)
      /* first 240px-wide background tile starts at 200px from left edge,
         so two circles touch in the center */
      50vw/60vw
```
Even CSS nerds themselves have to learn something new constantly to stay in the leaderboads, and apparently there is no result in CSSBattle that can't be improved :) so good luck and feel free to ask any questions about any solution here and in the Discord chats!
